src/Toggle.css

```diff
@@ -0,0 +1,27 @@
+.room-info {
```

Comment:
A lot of classes are unused in this -- this looks like a copy of RoomInfo?

Reply:
It doesn't look like toggle.js uses any of the classes in its css file, should I remove Toggle.css entirely?

Reply:
Yea, use RoomInfo.css instead as that handles the `color: white` case

Reply:
Removed Toggle.css and imported RoomInfo.css instead. Should appear in my next commit.